tests/paint_test.py

```diff
@@ -192,3 +192,32 @@ def test_gettransform(input_point, paint, expected_point):
 )
 def test_transformed(transform, target, expected_result):
     assert transformed(transform, target) == expected_result
+
+
+@pytest.mark.parametrize(
+    "transform, expected",
+    [
+        (
+            Affine2D(2, 0, 0, 1, 0, 0),
```

Comment:
Commenting the test cases would make this much more readable imo

Reply:
I simply moved the existing test_decompose_uniform_transform from another test module (since I have moved the tested function to paint.py where it's used).
I'm not sure what I should write in the comment. Maybe if I split the fixture `expected` into two values `uniform_transform` and `remaining_transform` would be clearer?